app/src/main/java/com/telex/review/InAppReviewManager.kt

```diff
@@ -23,8 +20,9 @@ class InAppReviewManager constructor(
 ) : AppReviewManager {
 
     override fun tryRequestAppReview(activity: Activity): Completable {
-        val activityReference = WeakReference(activity)
```

Comment:
@Mazide Please, bring back the weak reference. It's needed to avoid a memory leak.